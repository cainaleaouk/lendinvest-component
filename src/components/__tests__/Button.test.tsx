import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from '../Button';

describe('<Button /> snapshot', () => {
    it('should render correctly', () => {
        const component = renderer.create(
          <Button onClick={jest.fn}>Click me!</Button>
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
})

