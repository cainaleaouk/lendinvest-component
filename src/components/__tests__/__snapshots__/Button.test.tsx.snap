// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> snapshot should render correctly 1`] = `
<button
  className="sc-hKgILt eIFqbd"
  onClick={[Function]}
>
  <p
    className="sc-bdfBwQ sc-eCssSg hyyzsn bZQIWy"
  >
    Click me!
  </p>
</button>
`;
